dictionary-app/pages/index.js

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import axios from "axios";
+
+export default function Home() {
+  const [word, setWord] = useState("");
+  const [wordInfo, setWordInfo] = useState(null);
+
+  /**
+   *
+   *
+   * Fetch word information
+   */
+  const fetchInfo = async () => {
+    try {
+      const res = await axios.get(`/api/info`, {
+        params: { word },
+      });
+      const { data } = res;
+      const { definition } = data;
+
+      const newDefinition = definition.split(/1. |2. | 3. /);
```

Comment:
If you are describing why this is necessary, it's fine. It would be better to add a comment in the code as well.